SwiftWisdom/Core/Version/Version.swift

```diff
@@ -33,45 +33,47 @@ extension Version : CustomStringConvertible {
     }
 }
 
-extension Version : Equatable {}
+// MARK: Operator Overloads
 
-public func ==(lhs: Version, rhs: Version) -> Bool {
-    guard lhs.major == rhs.major else { return false }
-    guard lhs.minor == rhs.minor else { return false }
-    guard lhs.patch == rhs.patch else { return false }
-    return true
-}
+extension Version : Equatable {
+    public static func ==(lhs: Version, rhs: Version) -> Bool {
+        guard lhs.major == rhs.major else { return false }
+        guard lhs.minor == rhs.minor else { return false }
+        guard lhs.patch == rhs.patch else { return false }
+        return true
+    }
 
-public func !=(lhs: Version, rhs: Version) -> Bool {
-    return !(lhs == rhs)
+    public static func !=(lhs: Version, rhs: Version) -> Bool {
+        return !(lhs == rhs)
+    }
 }
 
-extension Version : Comparable {}
-
-public func < (lhs: Version, rhs: Version) -> Bool {
-    if lhs.major != rhs.major {
-        return lhs.major < rhs.major
-    } else if lhs.minor != rhs.minor {
-        return lhs.minor < rhs.minor
-    } else {
-        return lhs.patch < rhs.patch
+extension Version : Comparable {
+    public static func < (lhs: Version, rhs: Version) -> Bool {
+        if lhs.major != rhs.major {
+            return lhs.major < rhs.major
+        } else if lhs.minor != rhs.minor {
+            return lhs.minor < rhs.minor
+        } else {
+            return lhs.patch < rhs.patch
+        }
     }
-}
 
-public func <= (lhs: Version, rhs: Version) -> Bool {
-    return lhs < rhs || lhs == rhs
-}
+    public static func <= (lhs: Version, rhs: Version) -> Bool {
+        return lhs < rhs || lhs == rhs
+    }
 
-public func >= (lhs: Version, rhs: Version) -> Bool {
-    return lhs > rhs || lhs == rhs
-}
+    public static func >= (lhs: Version, rhs: Version) -> Bool {
+        return lhs > rhs || lhs == rhs
+    }
 
-public func > (lhs: Version, rhs: Version) -> Bool {
-    if lhs.major != rhs.major {
-        return lhs.major > rhs.major
-    } else if lhs.minor != rhs.minor {
-        return lhs.minor > rhs.minor
-    } else {
-        return lhs.patch > rhs.patch
+    public static func > (lhs: Version, rhs: Version) -> Bool {
+        if lhs.major != rhs.major {
```

Comment:
can this not be reduced to:
`return lhs.major > rhs.major  || lhs.minor > rhs.minor || lhs.patch > rhs.patch`